drivers/Balboa/device.js

```diff
@@ -309,12 +309,15 @@ module.exports = class device_Balboa extends Homey.Device {
             await this.setValue('measure_runmode', runModeReady, check);
 
             if (currentTemp) await this.setValue('measure_temperature', toCelsius(currentTemp), check, 10, settings.round_temp);
-            // If desiredTemp is available, compare it to targetDesiredTemp. There should be 0.4 difference for valid value.
+
+						// If desiredTemp is available, compare it to targetDesiredTemp. There should be 0.4 difference for valid value.
             // Use also desiredTemp when targetDesiredTemp is at highRangeHigh or lowRangeLow, when tempRange was changed.
-            // Fallback to targetDesiredTemp and helps desireTemp is not available or update is delayed in the device API.
-            // Values neet to be Number for the strict comparison.
-            targetDesiredTemp = Number(targetDesiredTemp);
-            desiredTemp = Number(desiredTemp);
+            // Fallback to targetDesiredTemp if desiredTemp is not available or update is delayed in the device API.
+            // Values need to be Number for the strict comparison.
+						// 2023-12-19 CMS API introduced a bug in numeric values where the decimal separtor is a comma instead of a dot.
+
+						targetDesiredTemp = Number(targetDesiredTemp.toString().replace(',', '.'));
+						desiredTemp = Number(desiredTemp.toString().replace(',', '.'));
```

Comment:
My guess would be that a parseFloat would also work here. But good for now :)

Reply:
Yep, quick fix indeed.